### ISSUE TITLE
Can't opt-out of lockdown mode (tips in bug report)

### ISSUE THREAD

**Issue by orazioedoardo** [blinksh/blink#1893]
### Checklist

- [x] Using latest version on the [App Store](http://itunes.apple.com/app/id1156707581)
- [x] Read the [docs](http://blink.sh/docs/)
- [x] Searched for [existing GitHub issues](https://github.com/blinksh/blink/issues)

### Configuration

Lockdown mode enabled.

### Describe the bug

Blink can't opted out of lockdown mode, comment https://github.com/blinksh/blink/issues/1564#issuecomment-1791600282 shows how it could be fixed. While I understand that lockdown mode means some features may not work, it is also supposed to be configurable on a per-app or per-site basis. [Lockdown mode is a very good bundle of security features in general, not as extreme as Apple paints it ](https://www.glitchcat.xyz/p/four-things-to-know-about-lockdown) and users may want to use Blink effectively with it without disabling the protection as a whole.

**Comment by carloscabanero** [comment]
Thanks for the extra information on this. I will need to take a deeper look and thanks for sending the article. Help me out though, the general idea is that we would allow the user to have Blink outside lockdown mode when lockdown is enabled, while we keep the other tied up version in other cases, correct?

**Comment by orazioedoardo** [comment]
> the general idea is that we would allow the user to have Blink outside lockdown mode when lockdown is enabled

Yes, Blink would be excluded while other apps and websites would not. While testing the feature, apps empirically appear with a toggle in Settings -> Privacy & Security -> Configure Web Browsing.

> we keep the other tied up version in other cases

Which other version?

**Comment by jayhemsley** [comment]
+1 would want this feature, with lockdown mode enabled VSCode doesn’t work properly.
